lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -315,6 +315,8 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
                 address=ip_addr, username=username, password=password
             )
 
+            self._expand_root_partition(node)
```

Comment:
Add a comment here explaining why we need this even though we're doing `_resize_vhd_if_needed()` above.